libs/form/src/next/input-field-masked/stories/vl-input-field-masked.stories-doc.mdx

```diff
@@ -36,10 +34,93 @@ Bij het `price` mask wordt de prefix weggelaten.
 
 ## Validatie
 
-- Bij het gebruik in een form wordt er automatisch mask validatie toegevoegd aan het input veld.
-- De `patternMismatch` ValidityState key wordt gebruikt voor de mask validatie.
-- Het 'pattern' attribuut wordt uitgeschakeld omdat het mask deze werking overneemt, gelieve dit attribuut niet te gebruiken.
-- Het is mogelijk om de mask validatie te combineren met andere validaties (bv. `required`).
+- Er wordt automatisch mask validatie toegevoegd aan het input veld, je kan dit uitschakelen met het `disable-mask-validation` attribuut.
+- Bij de mask validatie wordt er gebruik gemaakt van een regex, deze kan je overschrijven met de `regex` property.
+- Bij het testen van de regex wordt altijd de raw value van het input veld gebruikt.
+- De `patternMismatch` ValidityState key wordt gebruikt voor de mask validatie error.
```

Comment:
Eventueel nog link naar https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/pattern toevoegen en/of ook expliciet vermelden dat dit `patternMismatch` ValidityState key gebruikt zal worden voor zowel `pattern` als `regex`-variant.

Reply:
De input-field-masked maakt enkel gebruik van de regex, pattern kan je hier niet gebruiken.
Om verwarring te vermijden plaats ik deze link er liever niet bij.